gef-ui/api-tests/common-tests/role-validation-api-tests.js

```diff
@@ -22,9 +22,9 @@ const withRoleValidationApiTests = ({
   makeRequestWithHeaders,
   whitelistedRoles,
   successCode,
-  successHeaders,
+  successHeaders = undefined,
   disableHappyPath = false, // TODO DTFS2-6697: remove and test happy paths.
-  redirectUrlForInvalidRoles,
+  redirectUrlForInvalidRoles = undefined,
```

Comment:
Why are these set to undefined now please?
 
`withRoleValidationApiTests` has conditional assertions with `successHeaders` at least, it looks like these will not run now in any tests

Reply:
These are just fallback values, so those tests will still run when e.g. `successHeaders` is passed in.

But I've reverted this file back and I'm instead now passing in 'null' as values for `successHeaders` and `redirectUrlForInvalidRoles` in the `what-needs-to-change.api-test` file 🙂 

How does that sound?